mon-pix/translations/en.json

```
@@ -1366,8 +1366,17 @@
       "first-title": "Oops, an error occurred!"
     },
     "evaluation-results": {
+      "leave-without-signup-modal": {
+          "actions": {
+            "cancel": "Cancel",
+            "leave": "Yes, leave the course"
+          },
+          "content-information": "If you leave this course without registering, you will not be able to validate your progress.",
+          "content-instruction": "Would you like to continue?",
+          "title": "Leave and return to pix.fr?"
```

Le titre ne devrait pas avoir `pix.fr` mais `Pix`, ça n'est pas le même domaine pour les différentes langues. (idem pour toutes les autres langues)

```suggestion
          "title": "Leave and return to Pix?"
```